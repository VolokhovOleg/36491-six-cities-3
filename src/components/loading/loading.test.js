import Loading from './loading';

it(`<Loading /> snapshot`, () => {
  const tree = renderer
    .create(<Loading />
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
