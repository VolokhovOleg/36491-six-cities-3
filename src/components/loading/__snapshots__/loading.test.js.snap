// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> snapshot 1`] = `
<h1
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "height": "100vh",
      "justifyContent": "center",
    }
  }
>
  Loadingâ€¦
</h1>
`;
